corehq/apps/domain/forms.py

```diff
@@ -1223,6 +1223,13 @@ def current_account(self):
     def current_subscription(self):
         return Subscription.get_subscribed_plan_by_domain(self.domain)[1]
 
+    @property
+    @memoized
+    def current_contact_emails(self):
+        return BillingContactInfo.objects.get_or_create(
+            account=self.current_subscription.account
+        )[0].emails
```

Comment:
i agree, this actually happens in a couple different spots in the billing code. gonna merge so you can iterate, but do think it's a good suggestion